### DISCUSSION TITLE
Stop Starting and Stopping Emails

### DISCUSSION THREAD

**Discussion by gregeeh** [fail2ban/fail2ban#3753] [Q&A] [answered]
Hi,

I have installed Vaultwarden and F2B in containers on my NAS and I'm accessing VW via Cloudflare Tunnel. Failed logins are being banned at the CF level and I'm getting email notifications of failed attempts.

The problem is I'm getting email notifications of the **F2B container being stopped, restarted or started**. Can these be stopped but still get the email notifications for failed login attempts?

Searching finds several suggestions but none when using CF.

My vaultwarden.local jail is below.

TIA

```
[vaultwarden]
enabled = true
port = 80,443,8081
filter = vaultwarden
banaction = %(banaction_allports)s
logpath = /remotelogs/vaultwarden/vaultwarden.log
maxretry = 3
bantime = 14400
findtime = 14400
chain = FORWARD
destemail = greg@domain.au
sender = greg@domain.net.au
sendername = Fail2ban
cfapikey = f720110xxxxxxxxxxxx241a7c476098bce8a
cfemail = greg@domain.net.au
action = %(action_cf_mwl)s
    iptables-allports
```

**Comment by sebres** [comment, marked as answer]
Yes, you can...
Basically one needs to overwrite `actionstart` and `actionstop` for the mailing action.
It's not obvious to me whether it'd work using `action_cf_mwl` substitution, because of multiple action definition in:
https://github.com/fail2ban/fail2ban/blob/304c3cd56693c9ba90ace8b11c656ac2912648ca/config/jail.conf#L241-L242
However you can try it (both params must be supplied to last action, which is mailing action normally):
```diff
- action = %(action_cf_mwl)s
+ action = %(action_cf_mwl)s[actionstart='', actionstop='']
```
If it'd not work (excepting once, after restart it would once sends the mail, because not yet reloaded),
either you'd create a local action file `action.d/*-whois-lines.local` (replace * by your mta used, default is sendmail, but it depends on distro/maintained code) and add there this:
```ini
[Definition]
actionstart =
actionstop =
```
Or in `jail.local` for jail or default you'd redefne the `action` similar to `action_cf_mwl`:
```
action = cloudflare[cfuser="%(cfemail)s", cftoken="%(cfapikey)s"]
         %(mta)s-whois-lines[sender="%(sender)s", dest="%(destemail)s", logpath="%(logpath)s", chain="%(chain)s", actionstart='', actionstop='']
```

**Reply by gregeeh** [reply]
Thanks for your suggestion.  It worked !!!!
